### ISSUE TITLE
Haagas verwijderen

### ISSUE THREAD

**Issue by GreenpointAdviesPN** [Stichting-CROW/imbor#1170]
Ik stel voor het Haagas objecttype te verwijderen. Het object Haag bestaat al en als je daarbij een lengte wilt hebben, kan dat gewoon in een lente-attribuut worden opgenomen of evt. in een informatieobject (gelijkend Rand Groenobject).
Er is nu geen enkele semantische relatie tussen Haagas en Haag, waardoor je in het beheer zult tegenkomen dat beide objecten niet gelijk beheerd worden.
Haagas heeft een andere toepassing en een ander gewicht dan bijv. Wegas en Waterloopas. Die laatste twee kunnen enkelvoudig geregeistreerd worden over meerdere objecten heen waarmee ze verband houden. Voor Haagas geldt dat niet.

**Comment by RiX012** [comment]
@wjtmollema dit kunnen we wellicht meenemen in #997

**Comment by wjtmollema** [comment]
Inderdaad; `Haagas` is net als `Wegas` en `Waterloopas` een geometrische representatie van een object, maar geen netwerkobject (zie https://github.com/Stichting-CROW/imbor/issues/997#issuecomment-1500327639).

Zoals wordt gesteld in de melding is `Haagas` alleen bruikbaar i.r.t. `Haag` en valt dit `ObjectType` eigenlijk onder de paraplu van met GIS-handelingen afleidbare gegevens van de gekozen geometrische representatie van `Haag`.

**Comment by wjtmollema** [comment]
Gezien de impact voor groenbeheer zoals dat met de huidige modellering mogelijk is, wordt er voorgesteld de uitwerking van deze issue door de Werkgroep Groen te laten geschieden. De argumentatie van de melder is zinnig, maar wellicht is het wijs dat inhoudelijke experts hierover het laatste woord hebben. Idem voor @RiX012 Akkoord?

**Comment by JochemMollema** [comment]
Haagas is een hulplijn voor het bepalen van de lengte. Het objecttype Haagas hoeft niet verplicht gebruikt te worden.

**Comment by JochemMollema** [comment]
@RiX012  deze issue kan gesloten worden.
